### DISCUSSION TITLE
Can we run an experiment on a remote machine (without CML)?

### DISCUSSION THREAD

**Discussion by haimat** [treeverse/dvc#8940] [Help] [answered]
Hello everyone, I have not used DVC so far, but it looks great so far from reading the docs and we want to give it a try. One question came up: We would like to work locally to write the ML training scripts etc., but execute the DVC experiments on a remote machine because of its GPUs. Do we need CML for that, or can we simply trigger a DVC experiment with a parameter or something to let it run on another machine?

**Comment by pmrowla** [comment, marked as answer]
This is not currently supported, the suggested way to run experiments remotely would be via CML

**Reply by haimat** [reply]
Thanks, I will have a closer look at CML then :+1:

**Comment by courentin** [comment]
What about the [remote executor feature](https://github.com/iterative/dvc/wiki/Remote-executors), is it still "under active development"?

**Reply by pmrowla** [reply]
Unfortunately it is not currently being developed. Work on remote execution in DVC was tabled indefinitely in favor of focusing on CI scenarios (via CML) for the time being

**Reply by courentin** [reply]
Good to know thanks!
